tests/regression/test_regression.py

```diff
@@ -214,25 +244,22 @@ class TestObjectDetection(BaseTest):
             extra_overrides={"trainer.max_epochs": "10"},
         )
         for idx in range(1, 4)
+    ] + [
+        DatasetTestCase(
+            name="pothole_medium",
+            data_root="pothole_medium",
+            data_format="coco",
+            num_classes=1,
+            extra_overrides={"trainer.max_epochs": "10"}
```

Comment:
Same question here as above. Is there a reason why max_epochs should be different for each task if it's checking simply to check whether there is regression?